### ISSUE TITLE
Ubuntu 16.04 failing to install

### ISSUE THREAD

**Issue by CrypticCube** [dotnet/cli#6184]
## Steps to reproduce
apt-get install dotnet-dev-1.0.1

## Expected  behavior
Install .NET Core

## Actual behavior

```
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 dotnet-dev-1.0.1 : Depends: dotnet-sharedframework-microsoft.netcore.app-1.0.4                                                                                                                                                              but it is not going to be installed
                    Depends: dotnet-sharedframework-microsoft.netcore.app-1.1.1                                                                                                                                                              but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

```
## Environment data
`dotnet --info` output: none, dotnet not installed

## Comments

Tried to install dependencies separately and got these errors.
![Errors](https://cdn.discordapp.com/attachments/81384956881809408/296480339222724608/dnetsu.png)

**Comment by livarcocc** [comment]
@leecow is this something you can help with?

**Comment by CrypticCube** [comment]
Fixed, my VPS didn't have universe in source.list

**Comment by YoEight** [comment]
I'm having this exact issue and I have universe in source.list. For a matter of fact. I did successfully install `dotnet-dev-1.0.1` 3 weeks ago but fail right now on a new machine.

**Comment by YoEight** [comment]
I'm also running on Ubuntu 16.04 by the way

**Comment by warnov** [comment]
@YoEight have you set up the correct apt-get feed for Ubuntu 16.04? It must be Xenial. If you are not sure, try this:


    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 417A0893
    sudo apt-get update

and try again installing dotnet-dev-1.0.1

**Comment by ghanima** [comment]
Not sure who maintains the install instructions on: https://www.microsoft.com/net/core#linuxubuntu

The same steps denoted by @YoEight  needs to be applied to **_linux mint_** as well. Unless someone is going to fix the package dependencies for trusty.

**Comment by mairaw** [comment]
@leecow can you review the instructions on the site? /cc @kendrahavens

**Comment by leecow** [comment]
I'll add a Mint 18 label next to 16.04 on the Ubuntu page to make that bit clear. 

As far as installing the various packages individually, it's not recommended that you do so. If you do give it a try, the host and shared framework packages must be installed in a particular order. 

* `dotnet-host`
* `dotnet-hostfxr-1.0.1` and/or `dotnet-hostfxr-1.1.0`
* `dotnet-sharedframework-microsoft.netcore.app-1.0.4` and/or `dotnet-sharedframework-microsoft.netcore.app-1.1.0`

There is no 'raw' sdk package at this time.

**Comment by NETESudarshan** [comment]
@leecow: Hello, I followed the instructions on https://www.microsoft.com/net/core#linuxubuntu for Ubuntu 16.04. I got the 1.0.4 SDK (LTS) installed. How do I install the current 1.1.2 SDK? should it  be 'yaketty main' instead? Its not very clear.

Also, from the instructions:
```Before you start, please remove any previous versions of .NET Core from your system by using this script.
To install .NET Core 1.1 on Ubuntu or Linux Mint, simply use apt-get.```

So, every time we install a different/new SDK (or runtime) on Ubuntu, do we **have** to remove previous versions? We don't have to do that on Windows. 

-thanks
